datamade/how-to
# Add documentation on database backups

## Overview

See title.

Handles #208.

### Notes

I ported docs on our legacy pattern straight from `deploy-a-site`. Thanks, @jeancochrane and @evz, for authoring them! One of many ways your impact is still felt and appreciated by your friends at DataMade on the daily. ☺️

## Testing Instructions

* Review the prose and confirm it's accurate, straightforward, and sensical.
* View the rendered Markdown and confirm it looks good.
* Confirm links resolve.
